src/core/prototypes/plot_segments/axis.ts

```diff
@@ -563,6 +621,70 @@ export class AxisRenderer {
     }
   }
 
+  public renderPolarRadialGridLine(x: number, y: number, innerRadius: number, outerRadius: number, startAngle: number, endAngle: number) {
+    const style = this.style;
+    if (style.gridlineStyle === "none") {
+      return;
+    }
+    const g = makeGroup([]);
+    const rangeMin = this.rangeMin;
+    const rangeMax = this.rangeMax;
+    const gridineArcRotate = 90;
+    const lineStyle: Style = {
+      strokeLinecap: "round",
+      strokeColor: style.gridlineColor,
+      strokeWidth: style.gridlineWidth,
+      strokeDasharray: strokeStyleToDashArray(style.gridlineStyle)
+    };
+    for (const tickPosition of this.ticks
+      .map((x) => x.position)
+      .concat([rangeMin, rangeMax])
+      ) {
+      const cos = Math.cos(((-tickPosition + gridineArcRotate) / 180) * Math.PI);
```

Comment:
common function for degreesToRadians

Reply:
I added the function and applied everywhere where we have `value / 180 * Math.PI`. @danmarshall  could your review the last comment? Thanks!